code/procs/jobprocs.dm

```diff
@@ -681,6 +681,11 @@
 					if (src.limbs.r_arm)
 						qdel(src.limbs.r_arm.remove(0))
 				boutput(src, "<b>Your singular arm makes you feel responsible for crimes you couldn't possibly have committed.</b>" )
+
```

Comment:
```suggestion

	// Special mutantrace items
```

Reply:
Such a dishonest way of upping your commit count!

Reply:
you don't need to add me as a co-author when merging